examples/pattern_rewriting.py

```diff
@@ -60,7 +60,7 @@ def get_rotary_model(bad_model=False):
 
 
 model = get_rotary_model()
-ir_model = oir.irbuilder.build_ir(model)
+ir_model = ir.serde.deserialize_model(model)
```

Comment:
More a naming question really ... whether an alternative name might be better than serialize/deserialize.

Reply:
Sure - do you have suggestions? Convert? Just thinking a name that is not overloaded would be preferable

Reply:
We can create a class that defines methods convert_model_from_proto/convert_model_to_proto etc.

Reply:
Yes, `convert_X_to_Y` is reasonable for specific X. If we want a flexible version (allowing multiple source types), something like `convert_to_Y` or just `to_Y` is also fine. 

Re. the naming convention (I saw the issue you created for this): using a vanilla `model` has some disadvantages (especially since a lot of pre-existing code will contain references to vanilla `model` meaning a model in proto format or some other representation). Creating a more specific name (like `model_ir`) is also useful, as it can be used unambiguously where desired.

For the convert methods, using a name like `convert_proto_to_ir` or `convert_model_proto_to_ir` might be preferable to `convert_proto_to_model` (at least, that's my personal take).